File: seed/java-sdk/exhaustive/flat-package-layout/src/main/java/com/snippets/Example75.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.endpoints.types.GetWithPathAndQueryParamsRequest;

public class Example75 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .params()
                .getWithPathAndQuery(
                        "param",
                        GetWithPathAndQueryParamsRequest.builder()
                                .query("query")
                                .build());
    }
}

File: seed/csharp-sdk/exhaustive/explicit-namespaces/Snippets/Example75.cs
using SeedApi;
using SeedApi.Endpoints.Params;

public partial class Examples
{
    public async Task Example75() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.GetWithPathAndQueryAsync(
            new GetWithPathAndQueryParamsRequest {
                Param = "param",
                Query = "query"
            }
        );
    }

}
